src/dags/sql/stroomstoringen.py

```diff
@@ -0,0 +1,71 @@
+# The data contains epoch times.
+# In order to translate it to a timestamp, the data is updated.
+CONVERT_DATE_TIME = """
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_datum_gemeld TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_datum_schatting TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_datum_eind TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN date_end TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_datum_change TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN storing_service_update TYPE varchar(50);
+ALTER TABLE {{ params.tablename }} ALTER COLUMN log_date TYPE varchar(50);
+
+UPDATE {{ params.tablename }}
+SET storing_datum_gemeld = to_timestamp(rpad(storing_datum_gemeld, 13, '0')::bigint / 1000),
```

Comment:
Your comment says: `# The data contains epoch times.` if that is the case I think `to_timestamp(<column_name>)` is sufficient?

If it isn't sufficient could you then explain why you are using `rpad` and `/ 1000`?

Reply:
Good one, the dates are stored in millseconds. Unix timestamps measures time with seconds, and not milliseconds (in Postgres too). Therefore the values must be devided by 1000 before converting to timestamp. I wil add it to the comment :+1: